x-pack/legacy/plugins/ml/public/jobs/new_job_new/recognize/components/job_settings_form.tsx

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiAccordion,
+  EuiButton,
+  EuiDescribedFormGroup,
+  EuiFieldText,
+  EuiForm,
+  EuiFormRow,
+  EuiSpacer,
+  EuiSwitch,
+  EuiTextAlign,
+} from '@elastic/eui';
+import { JobGroupsInput } from '../../pages/components/job_details_step/components/groups/job_groups_input';
```

Comment:
I wonder if we should move this component to a place higher in the dir hierarchy when it's shared across pages like this now? (Applies to some other imports in this file too)